deploy.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
+
+export KRUIZE_UI_IMAGE="docker.io/bharathappali/kruize-ui:central-config"
+KRUIZE_UI_MANIFEST_FILE="kruize-ui-nginx-setup.yml"
+MANIFEST_TEMPLATE="./manifests/templates/kruize-ui-nginx-setup-template.yml"
+
+launch_to_kube_with_image=false
+is_image_set=false
+image_name=""
+
+POD_NAME="kruize-ui-nginx-pod"
+SERVICE_NAME="kruize-ui-nginx-service"
+NAMESPACE="monitoring"
```

Comment:
monitoring is only for minikube. For openshift, we are using openshift-tuning

Reply:
We have -e option which will set the namespace based on deployment platform